src/main/flight/position.c

```diff
@@ -219,28 +211,31 @@ void calculateEstimatedAltitude(void)
     DEBUG_SET(DEBUG_ALTITUDE, 3, estimatedVario);
 #endif
     DEBUG_SET(DEBUG_RTH, 1, lrintf(displayAltitudeCm / 10.0f));
+    DEBUG_SET(DEBUG_ALTHOLD, 1, lrintf(zeroedAltitudeCm));
+    DEBUG_SET(DEBUG_GPS_RESCUE_THROTTLE_PID, 1, lrintf(zeroedAltitudeCm));
 
     altitudeAvailable = haveGpsAlt || haveBaroAlt;
 }
+
 #endif //defined(USE_BARO) || defined(USE_GPS)
 
-bool isAltitudeAvailable(void) {
-    return altitudeAvailable;
+float getAltitudeCm(void)
+{
+    return zeroedAltitudeCm;
 }
 
-int32_t getEstimatedAltitudeCm(void)
+float getAltitudeDerivative(void)
 {
-    return lrintf(displayAltitudeCm);
+    return zeroedAltitudeDerivative; // cm/s
 }
 
-float getAltitude(void)
-{
-    return zeroedAltitudeCm;
+bool isAltitudeAvailable(void) {
+    return altitudeAvailable;
 }
 
-bool isAltitudeLow(void)
+int32_t getEstimatedAltitudeCm(void)
```

Comment:
This value is in most cases converted anyway. Returning `foat` in meters will only simplify code using it.

Reply:
True.  Let me look into how we best handle this low altitude value in a separate PR.
I will look at this if/when the 'low-altitude landing' process can be abstracted into one function in position_control.c, rather than having separate functions within GPS Rescue, failsafe landing, altitude_hold during active descent, and any 'landing mode' that we come up with, all doing much the same thing.